find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
+  xml_file = nil
+
+  files.each do |filename|
```

Comment:
I'm curious how the Dir.entries(folder) works. What order will the each loop evaluate the files in? This is for my own understanding of how this code works.